src/Routes.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Route, HashRouter, Switch } from 'react-router-dom'
+import OnboardingPage from './pages/onboarding/OnboardingPage'
+import OnboardingService from './services/OnboardingService'
+import FreshInstallPage from "../src/pages/fresh_install/FreshInstallPage";
+
+export default () => (
+    <HashRouter>
+        <Switch>
+            <Route exact path='/' component={getMainPage()} />
+            <Route exact path='/fresh-install' component={FreshInstallPage} />
+        </Switch>
+    </HashRouter>
+)
+
+const getMainPage = () => {
+    const onboardingService = new OnboardingService()
+    return onboardingService.isOnboardingFinished() ? FreshInstallPage : OnboardingPage
```

Comment:
@arctouch-jonathansouza @arctouch-caiodoneda Here we have a situation that me and @arctouch-alyssonlopes were discussing to see if we reach somewhere. Basically the question is "Does make sense to redirect the user directly to Fresh Install page if he already passed through onboarding page but closed the app ?"

Reply:
Yes, the onboarding page should be displayed only once